src/components/MeasurementOption/MeasurementOption.js

```diff
@@ -9,6 +9,11 @@ import selectors from 'selectors';
 
 import './MeasurementOption.scss';
 
+const DATA_ELEMENTS = {
+  SCALE_INPUT_CONTAINER: 'scaleInputContainer',
```

Comment:
added constants so its easier to work with rather than have hard coded strings everywhere